### ISSUE TITLE
Galactic Map Generator - Fix isolated star cluster on the galaxy map

### ISSUE THREAD

**Issue by MartinX3** [FAForever/gw-backend#20]
Isolated spots on the galaxy map needs to be fixed.
Steps to fix

1. Generate the Map
2. Iterate over faction less spots
3. Connect them with 3 of the next faction controlled spots.

**Comment by MartinX3** [comment]
@Brutus5000 
is that issue okay?

**Comment by Brutus5000** [comment]
Yes. As a general rule of thumb I'd say every disconnected subgraph should get connected via additional 3 edges minimum.

**Comment by MartinX3** [comment]
Updated.
